sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -9447,7 +9437,7 @@
                             "properties": {},
                             "hideLabel": true,
                             "disabled": true,
-                            "calculateValue": "value = utils.custom.currencyFormatter(data.disbursement?.disbursement1sbsd, '(Not eligible)');",
+                            "calculateValue": "value = utils.custom.currencyFormatter(data.disbursement?.disbursement1bcsg, '(Not eligible)');",
```

Comment:
This must be `disbursement1sbsd` i believe.

Reply:
Fixed. Thanks!